server/routes/api/messages.js

```diff
@@ -36,11 +36,27 @@ router.post("/", async (req, res, next) => {
       senderId,
       text,
       conversationId: conversation.id,
+      read,
     });
     res.json({ message, sender });
   } catch (error) {
     next(error);
   }
 });
 
+router.put("/:messageId", async (req, res, next) => {
```

Comment:
I think `/api/messages/:messageId/status` is a little more clear route name than `/api/messages/:messageId`. Would you update this one?

Reply:
I think here we are updating the `read` property of a resource i.e. `message`. I think that as per REST design api design principles, this how it should be. Because `status` it self is not a resource, it is rather a property of the resource (i.e. message)